client/css/style.css

```diff
@@ -2090,7 +2090,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
    For example: user has quit ( ===> L   O   L <=== )  */
 
 #windows .header .topic,
-#chat .message .text,
+#chat .message .text span,
```

Comment:
I did that at first, yeah, but that means that content of previews are also subject to that, plus do a `~` dance (or manually removing new lines, until next person breaks it) in the template... Unfortunately not ideal.